testdata/collection_v2.1.0.json

```
@@ -65,7 +65,8 @@
                 ],
                 "body": {
                     "mode": "raw",
-                    "raw": "{\"aKey\":\"a-value\"}"
+                    "raw": "{\"aKey\":\"a-value\"}",
+                    "options": "{\"raw\": {\"language\": \"json\"}"
```

This is why `TestWriteCollection` is failing. You wrote the `options` value as a string as it is an object.

```suggestion
                    "options": {
                        "raw": {
                            "language": "json"
                        }
                    }
```

---

done.